### DISCUSSION TITLE
Epilepsy warning lacks animation when you cancel loading the beatmap

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#15155] [Q&A] [answered]
for whatever reason, in the latest version of writing (2021.1016.0) the animation that normally happens when you cancel loading a beatmap, doesnt happen if you're on the epilepsy warning part of the beatmap-loading-thingy.

how to repro:
* find a map with an epilepsy warning (i used [this one](https://osu.ppy.sh/beatmapsets/20483#osu/71562))
* press enter
* when the epilepsy warning is showing, press esc
* ???? where the animation go?????

video: https://streamable.com/wh9bei
log: [runtime.log](https://github.com/ppy/osu/files/7359990/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15161
